subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -532,16 +532,26 @@ config BT_CTLR_LLCP_COMMON_TX_CTRL_BUF_NUM
 	  procedures/connections with allocation through a fifo queue.
 	  Configure between 0 and (4 - BT_CTLR_LLCP_PER_CONN_TX_CTRL_BUF_NUM) * BT_CTLR_LLCP_CONN
 
-config BT_CTLR_LLCP_PROC_CTX_BUF_NUM
-	int "Number of control procedure contexts to be available across all connections"
+config BT_CTLR_LLCP_LOCAL_PROC_CTX_BUF_NUM
```

Comment:
IIRC the reason for having the minimum number of contexts 2 so that we have at least 1 for local and 1 for remote procedures. Since we now have a separate pool for remote procedures the minimum can be set to 1 I think.
Sidenote: the way the unittests are written today they require a minimum of 4 local contexts available

Reply:
As discussed in meeting we keep the default value as was